README.md

```diff
@@ -58,6 +66,41 @@ spec:
             - Resource
 ```
 
+### Artifact Hub
+
+Add an `artifacthub-pkg.yml` metadata file to the folder. See an example metadata file for Kyverno policies below and customize per the comments.
+
+```yaml
+---
+name: backup-all-volumes # The name of the package (only alphanum, no spaces, dashes allowed)
+version: 1.0.0 # Version of the policy
+displayName: Backup All Volumes  # Display name of the policy
+createdAt: "2023-03-29T00:00:00.000Z" # The date this package was created (RFC3339 layout)
+description: >-
+# The description value should be taken from the relevant annotation policies.kyverno.io/description
+digest: 60ca548c88fc3e43db880bd5e466e4fa02af13b7f97c652aee59cb13ca9404e5 # The SHA256 hash String that uniquely identifies this package version
+install: |- # The installation instructions for the package
+    ```shell
+    kubectl apply -f https://raw.githubusercontent.com/kyverno/policies/main/velero/backup-all-volumes/backup-all-volumes.yaml
+    ```   
+keywords: # Keywords should always have "kyverno" and whatever the value of the policies.kyverno.io/category annotation. 
+  - velero
+  - kyverno
+readme: | # readme should be same as policies.kyverno.io/description annotation plus the last sentence as a static value.
+  In order for Velero to backup volumes in a Pod using an opt-in approach, it
+  requires an annotation on the Pod called `backup.velero.io/backup-volumes` with the
+  value being a comma-separated list of the volumes mounted to that Pod. This policy
+  automatically annotates Pods (and Pod controllers) which refer to a PVC so that
+  all volumes are listed in the aforementioned annotation if a Namespace with the label
+  `velero-backup-pvc=true`.
+
+  Refer to the documentation for more details on Kyverno annotations: https://artifacthub.io/docs/topics/annotations/kyverno/
+annotations: # Annotations values should be the same as policies.kyverno.io annotations and get annotations from https://artifacthub.io/docs/topics/annotations/kyverno/
```

Comment:
Change comment to:

"See the annotations guide on Artifact Hub here: https://artifacthub.io/docs/topics/annotations/kyverno/"